shuffle/cli/src/shared.rs

```diff
@@ -27,10 +27,17 @@ use url::Url;
 
 pub const MAIN_PKG_PATH: &str = "main";
 const NEW_KEY_FILE_CONTENT: &[u8] = include_bytes!("../new_account.key");
-pub const LOCALHOST_NETWORK_NAME: &str = "localhost";
-pub const LOCALHOST_NETWORK_BASE: &str = "http://127.0.0.1";
 const DIEM_ACCOUNT_TYPE: &str = "0x1::DiemAccount::DiemAccount";
 
+const LOCALHOST_NAME: &str = "localhost";
+const LOCALHOST_JSON_RPC_URL: &str = "http://127.0.0.1:8080";
+const LOCALHOST_DEV_API_URL: &str = "http://127.0.0.1:8081";
+
+const TROVE_TESTNET_NETWORK_NAME: &str = "trove";
+const TROVE_TESTNET_JSON_RPC_URL: &str = "http://trove.aws.hlw3truzy4ls.com";
+const TROVE_TESTNET_DEV_API_URL: &str = "https://api.trove.aws.hlw3truzy4ls.com";
+const TROVE_TESTNET_FAUCET_URL: &str = "https://trove.aws.hlw3truzy4ls.com/accounts";
```

Comment:
Why are we hardcoding these here vs reading from a config/env?

Reply:
These hard codings are written into the config once, and after that we only read from the config to get the Network values

Reply:
If that's the case, I'm a little confused why we don't just hardcode into the config and get rid of these?

Reply:
This is for config generation, not reading. On first run on `shuffle node`, we have an empty folder and we need a string literal to seed the first ~/.shuffle/Networks.toml with defaults, like localhost, so future runs can read from Networks.toml.